### PR TITLE
Add new removed functions and alternatives

### PR DESCRIPTION
**Pull request by sgchoi5**
Add new removed functions and alternatives in 5.5

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/545

**Comment by sgchoi5** [comment]
Done! Plz, review this change

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/545

File: docs/application/native/tutorials/details/native-removed-history.md [hunk 1]
```diff
@@ -33,6 +33,53 @@ The following table provides detailed information regarding removed functions an
...
+| Content - Media Content - Media Information | media_info_set_location_tag(), media_info_set_age_rating(), media_info_set_keyword(), media_info_refresh_metadata_to_db(), media_info_set_added_time(), media_info_create(), media_info_insert_to_db_with_data(), media_info_set_title() | Mobile, Wearable | Since 4.0 | 5.5 | No longer available | - |
+| Content - Media Content - Media Information | media_info_set_album(), media_info_set_artist(), media_info_set_genre(), media_info_set_recorded_date() | Mobile, Wearable | Since 4.0 | 5.5 | No longer available | - |
+| Content - Media Content - Media Storage | media_storage_get_name() | Mobile, Wearable | Since 4.0 | 5.5 | No longer available | - |
+| Content - Media Content - Media Information | video_meta_update_to_db() | Mobile, Wearable | Since 4.0 | 5.5 | No longer available | - |
+| Multimedia - Media Controller Client | mc_client_set_server_update_cb() | Mobile, Wearable | Since 4.0 | 5.5 | Better alternative s| mc_client_set_server_updated_cb() |
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/tutorials/details/native-removed-history.md]: There is no rule yet.
If a module has too many removed functions, separated them several parts.
If a removed function have alternative, let it have a line to emphasize alternative.

I've fixed typo.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/545

**Comment by sabeena21** [comment]
Hello @sgchoi5,
As per discussion with Annie/ Catherine following issues need to be addressed:

- Finding this specific page from navigation (on TD) is really tough.
https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/removed-functions-and-enumerations-tizen-native-api 
Can this page be referenced in the API reference home page, say "Removed/Deperacted APIs list"

- For the API names having "typo error", the corresponding "Alternative" column contains "New enums in the Application Framework" but with respect to "Better alternative" the exact replacement API names are specified.  Can we follow a uniform rule?

Example:
image
image


image
image


@annie-samsung @mijin85cho

**Comment by sgchoi5** [comment]
@sabeena21 
Regarding first issue. it needs to modify API Reference...... I will investigate it..
Regarding second issue. I will fix it to follow a uniform rule.

**Comment by sgchoi5** [comment]
done fixing table.
Regarding API Reference with deprecated/removed list, it is not related with this page.
I've made a test page for second issue. refer to my email.

**Comment by mijin85cho** [comment]
Another question : This PR is also considering alternative items in "5.5".
If so, isn't it better to work on "tizen_5.5_prepare" branch, not on master?

**Comment by sgchoi5** [comment]
This page was created for warning regarding API removal... So, It should be open earlier than 5.5 release.
master branch is correct.

File: docs/application/native/tutorials/details/native-removed-history.md [hunk 2]
```diff
@@ -12,27 +12,134 @@ The following table provides detailed information regarding removed functions an
...
+| Network - Bluetooth | bt_adapter_le_start_advertising() | Mobile | Since 2.3.1 | 5.0 | No longer available | - |
+| Network - Bluetooth | bt_gatt_foreach_primary_services() | Mobile | Since 2.3.1 | 5.0 | Better alternatives | bt_gatt_client_foreach_services or bt_gatt_client_get_service() |
+| Network - Bluetooth | bt_gatt_get_service_uuid() | Mobile | Since 2.3.1 | 5.0 | No longer available | - |
+| Network - Bluetooth | bt_gatt_foreach_included_services() | Mobile | Since 2.3.1 | 5.0 | Better alternatives | bt_gatt_service_foreach_included_services() or bt_gatt_service_get_included_service() |
+| Network - Bluetooth | bt_gatt_set_characteristic_changed_cb() | Mobile | Since 2.3.1 | 5.0 | Better alternative | bt_gatt_client_set_characteristic_value_changed_cb() |
+| Network - Bluetooth | bt_gatt_unset_characteristic_changed_cb() | Mobile | Since 2.3.1 | 5.0 | Better alternative | bt_gatt_client_unset_characteristic_value_changed_cb() |
+| Network - Bluetooth | bt_gatt_watch_characteristic_changes() | Mobile | Since 2.3.1 | 5.0 | Better alternatives | bt_gatt_client_set_characteristic_value_changed_cb() |
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/tutorials/details/native-removed-history.md]: Fixed

File: docs/application/native/tutorials/details/native-removed-history.md [hunk 3]
```diff
@@ -12,27 +12,134 @@ The following table provides detailed information regarding removed functions an
...
+| Content - Media Content | audio_meta_get_played_count(), audio_meta_get_played_time(), audio_meta_get_played_position(), audio_meta_set_played_count(), audio_meta_set_played_time(), audio_meta_set_played_position(), media_content_set_db_updated_cb(), video_meta_get_played_time(), video_meta_get_played_position(), video_meta_set_played_count(), video_meta_set_played_time(), video_meta_set_played_position()|Mobile, Wearable | Since 2.4 | 5.0 | Not available | - |
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/tutorials/details/native-removed-history.md]: Fixed

**Comment by sabeena21** [comment]
Sure @mijin85cho,  I will again review the native-removed-history.md file.

**Comment by mijin80lee** [comment]
TD Sync Completed.(reviewing state)

**Pull request merged**